gen/k5login_root

```diff
@@ -45,7 +45,7 @@ foreach my $resourceData (@resourcesData) {
 	local $, = "\n";
 	local $\ = "\n";
 
-	for my $destination (keys %outputByDestination) {
+	for my $destination (@generatedDataDestination) {
```

Comment:
This behavior will use only destinations from the last processed group, because it will always replace content of the array in the for-each cycle.

Reply:
Fixed